### DISCUSSION TITLE
Zigbee2mqtt problema linkquality mappa dispositivi con Sonoff  3

### DISCUSSION THREAD

**Discussion by Molot77** [indomus/forum#1648] [HUB: Home Assistant]
Ciao a tutti.
Oggi ho aggiornato il firmware del Sonoff zigbee dongle 3 all'ultima versione seguendo la guida di Indomus. e sull'interfaccia web di Zigbee2mqtt non si vede più il linkquality sulla tab della mappa, mi da tutti 0 quando prima funzionava tutto.
Nella tab cruscotto il linkquality si vede perfettamente ed i sensori funzionano a dovere ma sulla mappa nulla.
Soluzioni? 
Grazie

**Comment by snakuzzo** [comment]
Lascia perdere quella mappa...non è affidabilissima.
Poi aspetta qualche ora e ricontrolla.
Non pensare alla mappa...poniti il problema quando un device non funzionerà..finché va, stai sereno
